packages/app/src/specs/generators/EmptyGenerator.vue

```diff
@@ -56,6 +56,7 @@
       >
         <Button
           size="lg"
+          type="submit"
```

Comment:
I did not realize you could do fallthrough atrributes like this with Vue (https://vuejs.org/guide/components/attrs.html), but learned something new today!